www/src/components/Menu.tsx

```diff
@@ -1,18 +1,20 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 import { Link, graphql, useStaticQuery } from 'gatsby';
 import {
-  Container,
-  Row,
-  Col,
   Tooltip,
   OverlayTrigger,
-  Popover,
   Button,
+  Popover,
   Badge,
   Icon,
+  Collapsible,
+  Hyperlink,
 } from '~paragon-react';
-import { Issue } from '../../../icons';
+import { AddTask } from '../../../icons';
```

Comment:
nit: we have an alias import for icons, can be changed to `import { AddTask } from '~paragon-icons';`

Reply:
done